cypress/integration/unit_tests/manage_layers_test.js

```diff
@@ -198,9 +198,11 @@ describe('Unit tests for manage_layers page', () => {
     const tbody = createAndAppend('tbody', 'tbody');
     const rows = createTrs(2);
     tbody.append(rows);
+    const colorEditor = createAndAppend('div', 'color-fxn-editor').show();
 
     cy.stub(window, 'confirm').returns(true);
     cy.wrap(onDelete(rows[0])).then(() => {
+      expect(colorEditor.is(':visible')).to.be.false;
```

Comment:
Added this line here to test the color editor being hidden - lmk if you'd like more

Reply:
Oops, thanks!